packages/www/components/Register/index.tsx

```diff
@@ -9,14 +16,15 @@ import BroadcastingProvider from "../Login/BroadcastingProvider";
 // The frontend salts are all the same. This could be configurable someday.
 export const FRONTEND_SALT = "69195A9476F08546";
 
-const Login = ({ id, buttonText, onSubmit, loading, errors }) => {
+const Register = ({ id, buttonText, onSubmit, loading, errors }) => {
   const router = useRouter();
   const [firstName, setFirstName] = useState("");
   const [lastName, setLastName] = useState("");
   const [organization, setOrganization] = useState("");
   const [phone, setPhone] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [isCompanyAffiliated, setIsCompanyAffiliated] = useState(true);
```

Comment:
Especially if we move the fields to after user/password fields, would be much cleaner to default this to `false` IMO. Users might click it on automatic as if it were a terms of service thing and then they see the new field.